queries.sql

```diff
@@ -15,3 +15,44 @@ SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name != 'Gabumon';
 
 SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+/* Project Day 2 */
+
+UPDATE animals SET species = 'unspecified';
+
+ROLLBACK;
+
+UPDATE animals SET species = 'digimon' WHERE name like '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE species is NULL;
+
+DELETE FROM animals;
+
+rollback;
+
+SELECT * FROM animals;
+
+DELETE FROM animals WHERE date_of_birth >= '2022-01-01';
+
+SAVEPOINT first_savepoint;
+
+UPDATE animals SET weight_kg = weight_kg*-1;
+
+ROLLBACK TO SAVEPOINT first_savepoint;
```

Comment:
- [ ]  Kindly make sure you write here the exact command you run to make the task done. a transaction is supposed to start with a BEGIN.
```suggestion
BEGIN;
DELETE FROM animals WHERE date_of_birth >= '2022-01-01';

SAVEPOINT first_savepoint;

UPDATE animals SET weight_kg = weight_kg*-1;

ROLLBACK TO SAVEPOINT first_savepoint;
```